tests/Makefile

```diff
@@ -3,16 +3,21 @@ export CFLAGS+=-g -O0 -Wall -D_GNU_SOURCE
 
 DISTRO := $(shell ./os_detect)
 
+# Tests for all distributions.
 SUBDIRS := \
 	exec_execve \
-	exec_name \
 	file_create \
 	file_delete \
 	file_rename \
 	login_tty \
 	syscalls_file \
 	user_msg
 
+# Tests not suitable for RHEL6.
+ifneq ($(DISTRO), RHEL6)
+	SUBDIRS += exec_name
+endif
```

Comment:
I think it might be easier to list all the tests and then filter them out for each legacy distro we support, for example:

```
ifeq ($(DISTRO),RHEL6)
SUBDIRS:=$(filter-out exec_name, $(SUBDIRS))
endif
```